l10n_it_ricevute_bancarie/account/account.py

```diff
@@ -85,3 +87,28 @@ class account_invoice(osv.osv):
     _columns = {
         'unsolved_move_line_ids': fields.many2many('account.move.line', 'invoice_unsolved_line_rel', 'invoice_id', 'line_id', 'Unsolved journal items'),
         }
+
+    def action_cancel(self, cr, uid, ids, context=None):
+        if len(ids) > 0:
+            invoice = self.browse(cr, uid, ids)[0]
```

Comment:
Il 11/08/2015 16:16, Lorenzo Battistini ha scritto:
>
> In l10n_it_ricevute_bancarie/account/account.py 
> <https://github.com/OCA/l10n-italy/pull/128#discussion_r36749288>:
>
> > @@ -85,3 +87,28 @@ class account_invoice(osv.osv):
> >      _columns = {
> >          'unsolved_move_line_ids': fields.many2many('account.move.line', 'invoice_unsolved_line_rel', 'invoice_id', 'line_id', 'Unsolved journal items'),
> >          }
> > +
> > +    def action_cancel(self, cr, uid, ids, context=None):
> > +        if len(ids) > 0:
> > +            invoice = self.browse(cr, uid, ids)[0]
>
> I meant: when |len(ids)| is > 1, you have to perform the check for 
> every invoice in |ids|
> So: |for invoice in self.browse(cr, uid, ids, context=context)|
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/OCA/l10n-italy/pull/128/files#r36749288>.
>
Now should be ok

-- 
Andrea Cometa - Consulente ERP, Coord Reg SIPSI
Presidente Associazione Odoo Italia
Via Platone, 24/A - 74023 - Grottaglie (TA) - Italy
Cell: 3389553466   Tel: 0999870931  Fax: 0999870217
info@andreacometa.it     http://www.andreacometa.it
pec: andrea.cometa@pec.it
linkedin: http://it.linkedin.com/in/andreacometa